internal/provider/provider.go

```diff
@@ -81,6 +81,9 @@ func (p *provider) Configure(ctx context.Context, req tfsdk.ConfigureProviderReq
 
 	config := dvc_mgmt.NewConfiguration()
 	config.AddDefaultHeader("Authorization", p.AccessToken)
+	config.AddDefaultHeader("dvc-referrer", "terraform")
+	metadata := fmt.Sprintf("{\"go_version\": \"%s\", \"terraform_version\": \"%s\"}", runtime.Version(), req.TerraformVersion)
```

Comment:
Go version here may not be needed - as these terraform modules/providers are provided in a compiled binary context. The better facet here would be the provider version.

Reply:
I will consult with the right people - this was explicitly given on the ticket to be the data to collect, but I'll check the reasoning and share your recommendation!